a11y.js

```diff
@@ -37,6 +37,7 @@ const pageTypes = {
     '/html/head/iframe',
     "//div[@id='root']/main/div/div/div/div/iframe",
   ],
+  photoGalleryPage: ['/html/head/iframe'],
```

Comment:
I don't think this is wrong, but what does `'/html/head/iframe'` do? It seems to mention some exclusion for mpulse above?

Reply:
yeah @andrewscfc I think so too and also from the [bbc-a11y docs](https://github.com/bbc/bbc-a11y/blob/master/guides/using/using-bbc-a11y-in-your-project.md#hiding-errors) there's a section about doing this to hide specific errors from elements which are bound to fail a11y checks

Reply:
Ah makes sense looking again with the docs, looks good